packages/rainbowkit/src/wallets/walletConnectors/browserWallet/browserWallet.ts

```diff
@@ -1,38 +1,10 @@
-import type { InjectedConnectorOptions } from '@wagmi/core/dist/connectors/injected';
-import { InjectedConnector } from 'wagmi/connectors/injected';
-import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
 import { Wallet } from '../../Wallet';
+import { getDefaultInjectedConnector } from '../../getInjectedConnector';
 
-/**
- * @protected `browserWallet` interface
- */
-export interface BrowserWalletOptions {
-  chains: Chain[];
-}
-
-/**
- * @protected eip-1193 wallet connector
- */
-export const browserWallet = ({
-  chains,
-  ...options
-}: BrowserWalletOptions & InjectedConnectorOptions): Wallet => ({
-  id: 'browser',
+export const browserWallet = (): Wallet => ({
+  id: 'injected',
   name: 'Browser Wallet',
   iconUrl: async () => (await import('./browserWallet.svg')).default,
   iconBackground: '#fff',
-  hidden: ({ wallets }) =>
-    wallets.some(
-      (wallet) =>
-        wallet.installed &&
-        wallet.name === wallet.connector.name &&
-        (wallet.connector instanceof InjectedConnector ||
-          wallet.id === 'coinbase'),
-    ),
```

Comment:
Because `hidden` is removed here, we'll need to do this check in `connectorsForWallets` and append `browserWallet` to the list there (as they were removed from `getDefaultWallets`. We should append `safeWallet` and `braveWallet` in the same way

Reply:
It's actually super difficult to do this check unless we try to use `createConfig` inside `connectorsForWallets`. Maybe that's the the only way, but we can't manually get `wagmi`s connector metadata since it returns `CreatConnectorFn`

Reply:
We can probably do this in the react tree to so we can pass in `config.interal._connectors` and then pass into our own function again to see wether we should append `browserWallet`, `safeWallet`, `braveBrowser` and etc...

Reply:
Also not sure if we should append it in another category group in connect modal